examples/library/test_printing/main.c

```diff
@@ -50,6 +50,7 @@
 #include <time.h>
 #include <stdlib.h>
 #include <bsg_manycore.h>
+#include <bsg_manycore_printing.h>
```

Comment:
why is this here?

Reply:
Fails silently with a linker error if it isn't.

Reply:
I'll pull it out into a separate PR